tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import wknml
+import pickle
+from tests.test_snapshot_readandwrite import INPUT_FILES, OUTPUT_FILES, SNAPSHOT_FILES
+
+
+def __save_snapshot_pickle_read_and_write():
+    for i in range(0, len(INPUT_FILES)):
+        input_file = INPUT_FILES[i]
+        output_file = SNAPSHOT_FILES[i]
+        parsed = wknml.parse_nml(input_file)
+        with open(output_file + '.pickle', 'wb') as f:
+            pickle.dump(parsed, f)
+
+
+def __save_snapshot_nml_read_and_write():
```

Comment:
Why the `__` prefix?

Reply:
so it doesn't get executed by pytest. Alternatively I could give pytest as a parameter the test file to test

Reply:
Changed it.

Reply:
Ah, I thought pytest would only execute test functions which are prefixed with "test". Nevermind!